### PR TITLE
fix(db): The method of determining whether opening the global client session is incorrect

### PR DESCRIPTION
**Pull request by zijiacj**
#### What type of PR is this?
type-bug

#### What this PR does / why we need it:
The current method for determining whether a global session is enabled applies only to `system` tenants, but not to `mysql` and `oralce` tenants. Instead, should adopt this way that determining whether the result set after running `show processlist` contains the column `time`.
#### Which issue(s) this PR fixes:

Fixes #

#### Additional documentation e.g., usage docs, etc.:

```docs

```

### CODE REVIEW

File: server/odc-service/src/main/java/com/oceanbase/odc/service/db/session/DefaultDBSessionManage.java [hunk 1]
```diff
@@ -300,38 +301,18 @@ private boolean isGlobalClientSessionEnabled(ConnectionSession connectionSession
...
+        // Check whether the global session is open
+        // If the global session is open, the "time" column will be displayed in the result set after
+        // executing the sql statement of "show processlist"
+        return connectionSession.getSyncJdbcExecutor(ConnectionSessionConstants.BACKEND_DS_KEY)
+                .query("show processlist", rs -> {
+                    try {
+                        int columnIndex = rs.findColumn("time");
+                        return columnIndex > 0;
+                    } catch (SQLException e) {
+                        return false;
```
Comment (PeachThinking) [inline server/odc-service/src/main/java/com/oceanbase/odc/service/db/session/DefaultDBSessionManage.java:313]: Is it necessary to add a log to print the cause of the exception in this code?

Comment (zijiacj) [reply server/odc-service/src/main/java/com/oceanbase/odc/service/db/session/DefaultDBSessionManage.java:313]: ok

File: server/odc-service/src/main/java/com/oceanbase/odc/service/db/session/DefaultDBSessionManage.java [hunk 2]
```diff
@@ -300,38 +301,18 @@ private boolean isGlobalClientSessionEnabled(ConnectionSession connectionSession
...
-                    .query("show proxyconfig like '" + configName + "';",
-                            rs -> rs.next() ? rs.getInt("value") : null);
-        } catch (Exception e) {
-            log.warn("Failed to obtain the value of OBProxy's configuration variable: {}", e.getMessage());
-            return null;
-        }
+        // Check whether the global session is open
+        // If the global session is open, the "time" column will be displayed in the result set after
+        // executing the sql statement of "show processlist"
+        return connectionSession.getSyncJdbcExecutor(ConnectionSessionConstants.BACKEND_DS_KEY)
```
[the remark this answers is not in the document]
Comment (zijiacj) [reply server/odc-service/src/main/java/com/oceanbase/odc/service/db/session/DefaultDBSessionManage.java:307]: yes

**Review by PeachThinking — APPROVED** [review]
lgtm

**Review by LioRoger — APPROVED** [review]
lgtm

**Pull request merged**